doc/classes/Viewport.xml

```diff
@@ -527,21 +527,26 @@
 		</constant>
 		<constant name="DEBUG_DRAW_OVERDRAW" value="3" enum="DebugDraw">
 			Objects are displayed semi-transparent with additive blending so you can see where they are drawing over top of one another. A higher overdraw means you are wasting performance on drawing pixels that are being hidden behind others.
+			[b]Note:[/b] When using this debug draw mode, custom shaders will be ignored. This means vertex displacement won't be visible anymore.
```

Comment:
"This means vertex displacement won't be visible anymore." Not sure it's easy to infer what "vertex displacement" means here. If it's not easy to understand this extra addendum only makes things sound harder than they are.

Reply:
It means both the `vertex()` and `fragment()` methods are replaced by the material override that the debug draw mode applies to all materials in the scene.